README.md

```diff
@@ -284,3 +285,21 @@ given our lack of previous experience in such a scenario. For this we are proud.
     - Meeting moderation and adgenda creation
     - Author of the PUM function (code review by Anders)
 
+## Building
+
+For building and running this project the requirements are:
+
+- [Maven v3.8.6](https://maven.apache.org/docs/3.8.6/release-notes.html)
+- Java 19+
+
+### How to build and run
+
+1. Clone this repository.
+2. Import project as a Maven project with your desired IDE.
+3. Run `Main.java`.
+
+Or by manually running with maven:
+
+1. Compile with `mvn compile`
```

Comment:
steps before step 1: 
- clone repo
- cd to root `cd decide-g26`

Reply:
Or via your terminal:

1. Clone this repository.
2. Enter repository root `cd decide-g26`
3. Compile with `mvn compile`
3. Run the program `mvn exec:java -Dexec.mainClass=com.group26.Main`
4. Test the program `mvn test`

Reply:
Good change.